_episodes/01-shell-basics.md

```diff
@@ -227,20 +223,20 @@ unintentionally run a different version and end up with different results.
 > /usr/bin/afni
 > ~~~
 > {: .bash}
-> Do not mix up with `locate` command, which (if available) would
+> Do not mix this up with the `locate` command, which (if available) would
 > just find a file with that word somewhere in the file name/path. 
 {: .solution}
 
 
 > ## Beware of built-in commands
 >
 > Some commands might be implemented by shell itself, and their
-> implementation might differ from the one provided by some core
+> implementation might differ from the one provided by a core
 > set of tools.
 >
 > Note that `which` is not a built-in command in bash (but is in
-> zsh), so in bash you would not be able to "resolve" built-in
-> commands such as e.g. `pwd`.
+> zsh), meaning that in bash you would not be able to "resolve”
```

Comment:
keep original `"` doublequotes

Reply:
Your attention span is frighteningly detailed